templates/miq-template-ext-db.yaml

```diff
@@ -203,13 +203,25 @@ objects:
               storage: "${APPLICATION_VOLUME_CAPACITY}"
 
 
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      description: "Headless service for ManageIQ backend pods"
+      service.alpha.openshift.io/dependencies: '[{"name":"${NAME}","namespace":"","kind":"Service"}]'
+    name: ${NAME}-backend
+  spec:
+    clusterIP: None
+    selector:
```

Comment:
I think we don't want selectors here.

From the doc:

>For headless services that define selectors, the endpoints controller creates Endpoints records in the API, and modifies the DNS configuration to return A records (addresses) that point directly to the Pods backing the Service.

I don't think we need any of that because we won't actually be addressing this service.